src/style/site.css

```diff
@@ -944,12 +944,6 @@ div.toggle-arrow-container {
   text-align: center;
 }
 
-.input-group-addon-jquery {
```

Comment:
Was this just an extra class we're not using anymore?

Reply:
Nevermind. I see it's a dup.

Reply:
k